src/components/Fields/TextArea.story.js

```diff
@@ -53,21 +63,22 @@ _NB: label will not hide if there is no value._
     ),
 );
 
-const multilineText = `Lorem ipsum dolor sit amet, consectetur adipisicing elit.
-
-Velit ipsam, nesciunt doloribus, aspernatur a itaque assumenda odit alias rem dignissimos culpa error magnam veniam cupiditate repellendus nam id perferendis sit.
-
-Velit ipsam, nesciunt doloribus, aspernatur a itaque assumenda odit alias rem dignissimos culpa error magnam veniam cupiditate repellendus nam id perferendis sit.`;
-
 stories.add(
-    'Auto Height',
+    'Fixed Height',
     storyWrapper(
-        'hasAutoHeight allows the textarea to resize with the content.',
+        `
+hasFixedHeight stops the textarea from resizing as the content changes.
+
+The example below has a fixed height of 200px.
+
+_NB: this mode also enables the default textarea resize function._
```

Comment:
This is no longer true, now that you hid the native resize handle.

Reply:
Thank you